modules/core/src/main/scala/gem/enum/package.scala

```diff
@@ -30,4 +34,38 @@ package object enum {
         case SmartGcalType.NightBaseline => baseline(GcalBaselineType.Night)
       }
   }
+
+  @SuppressWarnings(Array("org.wartremover.warts.Throw"))
+  implicit class HalfCompanionOps(companion: Half.type) {
+
+    def unsafeFromInt(n: Int): Half =
```

Comment:
It would, but that's not the semantics I want. We want `unsafeFromInt(n).toInt == n` which wouldn't be the vase if we normalized `n`. It makes sense for angles but I don't think it makes sense for enumerated types.